rubin_sim/maf/mafContrib/__init__.py

```diff
@@ -4,6 +4,7 @@
 from .varMetrics import *
 from .varDepthMetric import *
 from .lssMetrics import *
+from .lssmetrics import *
```

Comment:
Wow, that is confusing because they look so alike. Can we just lift the single file out of the lssmetrics directory and put it in here?